docs/source/installation.rst

```diff
@@ -4,4 +4,4 @@ Installation
 
 At the command line::
 
-    $ pip install suitcase-json-metadata
+    $ pip install suitcase-json_metadata
```

Comment:
Are we sure about this? I think it should be names the same way as the project, i.e. `suitcase-json-metadata`.